delete/deleteWPInstall.bash

```diff
@@ -27,7 +28,19 @@ rm -f /etc/nginx/sites-enabled/$FQDN
 rm -f /etc/nginx/sites-available/$FQDN
 rm -f /etc/php5/fpm/pool.d/$FQDN.conf
 
+# Revoke certificate. Will try to revoke all certs. 
+ls $letsencryptFolder/archive/$FQDN | grep "cert" | while read certificate; do
+	echo "Revoking cert $certificate"
+	../letsencrypt/letsencrypt-auto revoke --cert-path $letsencryptFolder/archive/$FQDN/$certificate
+	echo ""
+done
+# Remove config for domain
+rm -f $letsencryptFolder/$FQDN.cli.ini
```

Comment:
So you're using it here because you don't want the script to stop if the files don't exist?